### DISCUSSION TITLE
does Jugglucco auto switch connections

### DISCUSSION THREAD

**Discussion by Npavkov** [j-kaltes/Juggluco#103] [Q&A]
Does juggluco when using GW4 watch as collector automatically switch connections (BT,WIFI,LTE) when out of range of the phone?

**Comment by j-kaltes** [comment]
When a WearOS watch is directly connected with the sensor it sends data to Juggluco on the companion phone via WIFI or Bluetooth. If WIFI doesn’t work it uses Bluetooth.
This has nothing to do with the phone getting out of reach.
If you have a LTE WearOS watch (in which you can put an esim) and you want to connect it over the internet without taking the phone with you, you have to set up this manually the same way you can connect two phones running Juggluco over the internet, see https://www.juggluco.nl/Juggluco/phones/index.html
If Juggluco on the the watch or phone connects to Juggluco or Juggluco server, it will use mobile data or WIFI depending on what works. Juggluco or Juggluco server connected to shouldn’t test on the IP in that case.
